chan_debouncer/chan_debouncer.go

```diff
@@ -0,0 +1,23 @@
+package chan_debouncer
```

Comment:
Let's move it from `chan_debouncer/chan_debouncer.go` -> `debounce/simple_debouncer.go
```suggestion
package debounce
```